test/generators/ModelGenerators.scala

```diff
@@ -37,6 +38,11 @@ trait ModelGenerators {
       Gen.oneOf(TaxYearSelection.values)
     }
 
+  val generatorListOfTaxYearSelection: Gen[Seq[TaxYearSelection]] =
+    Gen.choose(1, TaxYearSelection.values.size)
+      .flatMap(pick(_,TaxYearSelection.values))
+      .map(_.toSet.toSeq)
```

Comment:
toSet.toSeq?

Reply:
Updated to make the intent of the generator clearer, ie that we're trying to build a list of unique TaxYearSelection